src/pages/reservations/ReservationCard.js

```diff
@@ -36,17 +37,19 @@ const ReservationCard = ({
       <p>
         <strong>Start Time: </strong>
         {' '}
-        {startTime}
+        {startTime.slice(0, 10)}
       </p>
       <p>
         <strong>End Time: </strong>
         {' '}
-        {endTime}
+        {endTime.slice(0, 10)}
       </p>
       <p>
         <strong>Duration: </strong>
         {' '}
         {duration}
+        {' '}
+        hours
```

Comment:
I think this is wrong. From what I can see in the backend code:

6.times do |i|
  PlaneReservation.create({ date: Date.today,
  start_time: Time.now,
  end_time: Time.now + 2.hours, user_id: 2,
  duration: 10, # here should be the difference between start_time and end time. It may be hours, mins, or even days so fix this
  plane_id: i })
end

Reply:
I don't know if you notice this: All of them are same and I feel like I cannot even look at which one is deleting. So Kindly add name of the plane
![Screenshot from 2023-04-06 22-28-57](https://user-images.githubusercontent.com/105173981/230435602-208522a8-4e6f-4154-98a9-b2c42097b662.png)

Reply:
Hi @heintayzar-hm, this is just a dummy data, in real world all the of this data would different from each other

Reply:
THe thing is you can look at the one which planes is client canceling